scripts/local/apply-all-migrations.sh

```
@@ -16,9 +41,9 @@ update_database() {
 
     if [ -f $env_path ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)